### ISSUE TITLE
Add JSCS etc.

### ISSUE THREAD

**Issue by LOZORD** [Beg-in/gulp-begin#2]
Do we want to add the same setup as https://github.com/Beg-in/Fritz/issues/4 ?

**Comment by BrainBacon** [comment]
Yeah let's just implement it here and use the implementation from here in Fritz.

**Comment by LOZORD** [comment]
I just gave JSCS (and gulp-jscs) a shot, and it keeps throwing an error: `TypeError: Cannot convert undefined or null to object`. I don't really know what the problem is :confused:

**Comment by BrainBacon** [comment]
Could you push your code to a new branch and I can try to diagnose it?

**Comment by LOZORD** [comment]
Here's the commit where I attempted to add it:

https://github.com/Beg-in/gulp-begin/commit/312295573b417f949992725b698b622f3f6fc7c0

I thought it would be pretty cut and dry, but it fails when I get to `gp.jscs()` (around line 424).
